### ISSUE TITLE
There is an extraneous note in the WCAG2ICT guidance on the definition of "structure" 

### ISSUE THREAD

**Issue by maryjom** [w3c/wcag2ict#187]
It seems there is a note that doesn't belong on the [definition of "structure"](https://wcag2ict.netlify.app/#guidance-when-applying-structure-to-non-web-documents-and-software) and should be deleted. This existed in the 2013 Note, but wasn't caught. This note talks about "AccessibleRole" and nothing in the definition of structure talks about roles, programmatic roles, etc.

>NOTE
>“AccessibleRole” (or the corresponding term used in different APIs) of the Accessibility API of the platform is an example of such a role.

**Comment by maryjom** [comment]
Definition for "structure" guidance has been approved by the AG WG. See Issue #270. No comments received.
